tests/sentry/tasks/integrations/github/test_pr_comment.py

```diff
@@ -508,3 +523,141 @@ def test_comment_workflow_missing_integration(
         mock_metrics.incr.assert_called_with(
             "github_pr_comment.error", tags={"type": "missing_integration"}
         )
+
+
+@region_silo_test(stable=True)
+class TestCommentReactionsTask(GithubCommentTestCase):
```

Comment:
Can you add a test case for if there are no reactions on a comment?

Reply:
if there are no reactions it's the same response -- example: https://api.github.com/repos/getsentry/sentry/issues/comments/1608494462